src/main/java/com/gregtechceu/gtceu/utils/fakelevel/BlockHighlight.java

```diff
@@ -110,8 +111,8 @@ public void renderSolid(Direction direction, PoseStack poseStack) {
 
     protected static void renderFrame(@Nullable Direction side, Float d) {
         if (side == null) {
-            for (int i = 0; i < 6; i++) {
-                buildFrameFace(side, d);
+            for (Direction direction : Direction.values()) {
```

Comment:
you can use `GTUtil.DIRECTIONS` here to avoid an array copy
```suggestion
            for (Direction direction : GTUtil.DIRECTIONS) {
```